### PR TITLE
Add support for unordered collections

### PR DESCRIPTION
**Pull request by tpendragon**

### CODE REVIEW

File: lib/hydra/pcdm/models/concerns/pcdm_behavior.rb [hunk 1]
```diff
@@ -49,11 +53,11 @@ def ordered_object_ids
 
     def parents
       warn '[DEPRECATION] `parents` is deprecated in Hydra::PCDM.  Please use `member_of` instead.  This has a target date for removal of 10-31-2015'
```
Comment (mjgiarlo) [inline lib/hydra/pcdm/models/concerns/pcdm_behavior.rb]: Should the warning message also `s/member_of/ordered_by/`?

File: lib/hydra/pcdm/collection_indexer.rb [hunk 2]
```diff
@@ -2,7 +2,9 @@ module Hydra::PCDM
   class CollectionIndexer < ObjectIndexer
     def generate_solr_document
       super.tap do |solr_doc|
-        solr_doc['member_ids_ssim'] = object.member_ids
+        solr_doc['member_ids_ssim'] ||= []
```
Comment (mjgiarlo) [inline lib/hydra/pcdm/collection_indexer.rb]: The string/symbol `member_ids_ssim` is all over the place -- maybe this should be put behind a method somewhere. Also happy to be told, "that's a separate PR."

File: lib/hydra/pcdm/models/concerns/pcdm_behavior.rb (hunk 1 above)
Comment (tpendragon) [inline lib/hydra/pcdm/models/concerns/pcdm_behavior.rb]: Yup.

**Comment by mjgiarlo** [comment]
Done reviewing. Looks good to me. :+1:

File: lib/hydra/pcdm/collection_indexer.rb (hunk 2 above)
Comment (tpendragon) [inline lib/hydra/pcdm/collection_indexer.rb]: Ticket please.

Comment (mjgiarlo) [inline lib/hydra/pcdm/collection_indexer.rb]: k

Comment (mjgiarlo) [inline lib/hydra/pcdm/collection_indexer.rb]: https://github.com/projecthydra-labs/hydra-pcdm/issues/189

**Pull request merged**